objcts.js

```diff
@@ -0,0 +1,10 @@
+let numbers = {}
+
+for (var number = 1; number <= 15; number++) {
+  var myKey = number
```

Comment:
We don't need this variable, See[ this conversation](https://github.com/SeR4h/andelaweek2challenges/pull/2#discussion_r349884399)